run_tests.py

```diff
@@ -270,6 +273,8 @@ def run_test(testname):
                 else:
                     if options.arch == 'x86':
                         gcc_arch = '-m32'
+                    elif options.arch == 'aarch64':
```

Comment:
Could you rework the condition starting on 271 line so it will be single condition with several elifs without nesting condition?